.claude/CLAUDE.md

```
@@ -0,0 +1,70 @@
+| local | 로컬 개발 | localhost MySQL |
+| dev | 개발 서버 | RDS (dev) |
+| prod | 운영 서버 | RDS (prod) |
+| test | 테스트 | TestContainers |
+
+## 명령어
+- 빌드: `./gradlew build`
+- 테스트: `./gradlew test`
+- 단일 테스트: `./gradlew test --tests "TestClassName"`
+- 로컬 실행: `./gradlew bootRun`
+- 코드 포맷팅: `./gradlew spotlessApply`
+- Swagger UI: http://localhost:8080/swagger-ui/index.html
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)